content/en/docs/guides/development/localization/localization-guide.md

```diff
@@ -42,6 +42,9 @@ Here's a detailed explanation.
 In order to get localized strings on your project, Strings need to be wrapped with a function appropriate to the file type.
 Please see the [Writing Localizable Code]({{< relref "writinglocalizableCode" >}}).
 
+##### note
```

Comment:
If you use note in the document, please use **{{< note >}} note content {{< /note >}}** style instead of "##### note".